metaci/repository/views.py

```diff
@@ -199,6 +200,21 @@ def github_push_webhook(request):
             branch.is_removed = False
             branch.save()
 
+    release = None
+    # Check if the event was triggered by a tag
+    if push['ref'].startswith('refs/tags/') and repo.release_tag_regex:
+        tag = push['ref'][10:]
+        # Check the tag against regex
+        if re.match(repo.release_tag_regex, tag) and push['head_commit']:
+            release, _ = Release.objects.get_or_create(
+                repo=repo,
+                git_tag=tag,
+                defaults = {
+                    'created_from_commit': push["head_commit"]["id"],
+                    'status': 'draft'
+                }
+            )
```

Comment:
It looks like if a tag is created and we create and publish a Release, and then the tag is force-changed to a different commit, that would trigger resetting the existing Release to draft status. I'm not sure we need to handle this case but maybe it would be better to abort if a release already exists for the tag?

Reply:
Oh yes, this is weird.

Reply:
Upon further examination and testing - this won't reset the existing Release in any way, but the new tag information will be thrown away.

Reply:
Okay, makes sense.